node-require/add.js

```diff
@@ -0,0 +1,5 @@
+const add = (x, y) => {
+    return parseInt(x)  +  parseInt(y);
+};

```

Comment:
What happens if someone tries to add two decimal numbers?

Reply:
I changed the parseInt to parsefloat . Tested the function.
![image](https://user-images.githubusercontent.com/20628519/127600453-f9ca9eaa-3c8d-4f82-90a0-0e12baf8a34a.png)